### PR TITLE
BUILD: pin imjoy-jupyterlab-extension for optional lab to 0.1.13

### PR DESCRIPTION
**Pull request by PaulHax**

### CODE REVIEW

**Review by thewtex — APPROVED** [review]

File: pyproject.toml [hunk 1]
```diff
@@ -66,7 +66,7 @@ all = [
     "aiohttp <4.0"
 ]
 lab = [
-    "imjoy-jupyterlab-extension >=0.1.13",
+    "imjoy-jupyterlab-extension ==0.1.13",
```
Comment (oeway) [inline pyproject.toml:69]: What's the reason to pin it to the old version? I thought we need that for the new jupyterlab. What did I missed?

Comment (PaulHax) [reply pyproject.toml:69]: I changed to follow this:
https://github.com/InsightSoftwareConsortium/itkwidgets/pull/689/commits/94fee5711ec87478d902a4c4b87047b08209ce48

Which references this issue that I have not checked myself:
https://github.com/imjoy-team/imjoy-jupyterlab-extension/issues/4

**Comment by oeway** [comment]
@PaulHax Could you try `imjoy-jupyterlab-extension==0.1.21`? I just fixed the issue.

**Comment by PaulHax** [comment]
> @PaulHax ~Could you try `imjoy-jupyterlab-extension==0.1.21`? I just fixed the issue.~
> 
> Edit: oops, it seems the new extension is not loading, I will see how I can fix it and get back to you.

Gotcha, thanks for taking a look.

**Pull request merged**